CHANGELOG.md

```diff
@@ -46,10 +49,7 @@
 - Added a new optional parameter called `format_json` to the `Session.SessionBuilder.app_name` function that sets the app name in the `Session.query_tag` in JSON format. By default, this parameter is set to `False`.
 
 #### Bug Fixes
-- Fixed a bug regarding precision loss when converting to Snowpark pandas `DataFrame` or `Series` with `dtype=np.uint64`.
```

Comment:
Should this be deleted?

Reply:
it's not showing up as deleted relative to main for me.